tests/dqe/dupchk.csv

```diff
@@ -0,0 +1,11 @@
+action,ms,bytes,lang,code,repeat,minver,comment
+comment,0,0,q,Start of dupchk function tests,1,,"comment"
+before,0,0,q,`.dqe.results insert (1 2 3 1 2 3;6#`.dqe.testing;6#`testsing;6#`dqetest;`rdb1`rdb1`hdb1`rdb1`rdb1`hdb1;raze(3#(.z.p-10:00);3#(.z.p-05:00));raze(3#(.z.p-09:00);3#(.z.p-04:00));111001b;6#enlist"testing";raze(3#`complete;`started`failed`complete)),1,,"Set up test table"
+before,0,0,q,.dqe.dupchk[1;`rdb1],1,,"Will fail the last id 1 that is still on started"
+before,0,0,q,.dqe.dupchk[1;`rdb2],1,,"Nothing happens as process is not in table"
+before,0,0,q,.dqe.dupchk[2;`rdb1],1,,"Nothin happens as las id 2 failed"
```

Comment:
Typo
```suggestion
before,0,0,q,.dqe.dupchk[2;`rdb1],1,,"Nothing happens as last id 2 failed"
```
Although I think the comment is a bit confusing anyway. Maybe something like:
> Nothing happens as last row for id 2 has failed

Also not sure if this should be a `before` test, maybe the check should be that this runs without fail? I think that is `run`.